### PR TITLE
TRP Response Message parser

### PR DESCRIPTION
**Pull request by TiagoGraf89**

### CODE REVIEW

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_113

File: Apps/Medication/src/Models/Prescription.cs [hunk 1]
```diff
@@ -38,12 +38,12 @@ public class Prescription
         /// <summary>
         /// Gets or sets the quantity for the prescribed prescription.
         /// </summary>
-        public int Quantity { get; set; }
+        public decimal Quantity { get; set; }
```
Comment (FuriousLlama) [inline Apps/Medication/src/Models/Prescription.cs]: Lets make it a float instead

File: Apps/Medication/src/Parsers/TRPMessageParser.cs [hunk 2]
```diff
@@ -117,8 +118,86 @@ public HNMessage CreateRequestMessage(string phn, string userId, string ipAddres
         /// <inheritdoc/>
         public List<Prescription> ParseResponseMessage(string hl7Message)
         {
+            List<Prescription> ret = new List<Prescription>();
```
Comment (FuriousLlama) [inline Apps/Medication/src/Parsers/TRPMessageParser.cs]: nit: ret-> prescriptions

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_113

**Comment by sslaws** [comment]
SonarQube Report: https://sonarqube-gateway.pathfinder.gov.bc.ca/dashboard?id=Medicationservice_pr_113

**Pull request merged**